_layouts/default.html

```diff
@@ -22,7 +22,7 @@
     {% include nav.html %}
   
     <main class="{% if page contains 'main_classes_override' %}{{ page.main_classes_override }}{% else %}flex-shrink-0 flex-grow-1 w-100{% endif %}" role="main">
-      <div class="container content-block d-block mx-auto pb-3">
+      <div class="container content-block d-block mx-auto pb-3 pxmd-4">
```

Comment:
This should be `px-md-4` else it has no effect.

Reply:
Damn thought i fixed that